ArduPlane/Attitude.cpp

```diff
@@ -587,6 +587,36 @@ void Plane::calc_nav_roll()
             plane.guided_state.last_forced_rpy_ms.x > 0 &&
             millis() - plane.guided_state.last_forced_rpy_ms.x < 3000) {
         commanded_roll = plane.guided_state.forced_rpy_cd.x;
+#if OFFBOARD_GUIDED == ENABLED
+    // guided_state.target_heading is radians at this point between equivalent of -180 and 180 degrees ( it also defaults to -1 )
```

Comment:
confusing comment, mixes radians and degrees on the same line, should be referred to as +/- PI here.

Reply:
will fix.